components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -520,6 +518,9 @@ const InsightPage = ({ edit, insight, pageRepos, workspaceId }: InsightPageProps
           </Title>
 
           <TextInput placeholder="Page Name (ex: My Team)" value={name} handleChange={handleOnNameChange} />
+          {validateName(name) ? (
```

Comment:
Thanks for the previous refactor. A couple things I noticed. The page shifts if the error message appears.

![CleanShot 2024-02-28 at 09 07 57](https://github.com/open-sauced/app/assets/833231/a4010f2f-6a86-4b65-a856-26294c85a8e8)

Always included the message, but using `validateName(name)` have it hidden or visible to avoid the jump in text and make the text `text-dark-red-8`.